### ISSUE TITLE
about:neterror has default styling

### ISSUE THREAD

**Issue by elerch** [overdodactyl/userContent-tweaks#11]
Thanks so much for putting this repo together!

Using FF 57.0.1 (64-bit) on macos El Capitan 10.11.6. I seem to remember originally seeing custom styling for about:neterror pages a couple days ago, but currently this appears to have the default styling. I usually test this by a totally random url in the address bar (e.g. adksjh.adfh). I do **not** see userContent.css show up in the style editor style list.

I played around the style editor adding a new style sheet with:

```css
@-moz-document url-prefix(about:neterror) {
  .neterror {background:black;}
}
```
and confirmed a background change. Doing so I was able to confirm that url-prefix is necessary (you have just "url"). It's unclear to me though why changing url to url-prefix in userContent.css from 57cb1be doesn't seem to pick up the changes after a restart.

**Comment by overdodactyl** [comment]
Hi @elerch,

Thank you for identifying/debugging this issue, it's really appreciated!

It appears having `url` only styled the page found when visiting `about:neterror` from the address bar, as opposed to the one you are normally shown.

As for needing a restart for that change to take place, I'm not entirely sure why that's required either..

But, either way, it should be fixed now :)

**Comment by overdodactyl** [comment]
Resolved - c7de63c

**Comment by elerch** [comment]
Looks great - thanks so much
